src/Timer/Timer.h

```diff
@@ -0,0 +1,111 @@
+/**@file
+ * This file is part of the Leon3 BSP for the Test Environment.
+ *
+ * @copyright 2022 N7 Space Sp. z o.o.
+ *
+ * Test Environment was developed under a programme of,
```

Comment:
This is something that was overlooked earlier. The license (ESA-PL) should be copied, but the part about funding should be removed.